ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json

```diff
@@ -265,6 +265,13 @@
     "CaseFieldID": "correspondenceDocumentsLANC",
     "UserRole": "[SOLICITOR]",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
from first file it looks that [SOLICITOR] should be able to create event, so not sure why case filed access is different that for other users that can create event

Reply:
supportingEvidence Documents can be updated in manage documents event so they will need `U` access

Reply:
question is why [SOLICITOR] does not have access to 
  {
    "LiveFrom": "01/01/2017",
    "CaseTypeID": "CARE_SUPERVISION_EPO",
    "CaseFieldID": "c2Type",
    "UserRole": "[LASOLICITOR]",
    "CRUD": "CRU"
  },
  like [LASOLICITOR] if both have right to create event

Reply:
oh, got it now.. I'll add the missing authorisation entries

Reply:
Added the missing entries for Solicitor role